### PR TITLE
Feature/deploy robot tool

### PR DESCRIPTION
**Pull request by jaagut**
## Necessary checks
- [ ] Update package version
- [ ] Run `catkin build`
- [ ] Write documentation
- [ ] Create issues for future work
- [ ] Test on your machine
- [ ] Test on the robot
- [ ] Put the PR on our Project board

### CODE REVIEW

File: scripts/deploy/deploy_robots.py [hunk 1]
```diff
@@ -0,0 +1,190 @@
...
+import argparse
+import os
+
+from rich.prompt import Prompt
+
+from deploy.misc import *
+from deploy.tasks import AbstractTask, AbstractTaskWhichRequiresSudo, Build, Configure, Install, Launch, Sync
+
+
+# TODO: If task arguments (e.g. -s) is given, only do given tasks. Do we wanna do this?
```
Comment (timonegk) [inline scripts/deploy/deploy_robots.py:12]: Yes. `-s` should only sync, `-b` should only build. That's how it was before and I think that is the most common use case.

File: sync_includes_wolfgang_nuc.yaml [hunk 2]
```diff
@@ -1,4 +1,5 @@
 include:
+    # TODO: Dont sync wolfgang pybullet sim, bitbots_ceiling_cam, wolfgang_robocup_api, webots?
```
Comment (timonegk) [inline sync_includes_wolfgang_nuc.yaml:2]:
```suggestion
```

File: scripts/robot_compile.py [hunk 3]
```diff
@@ -1,5 +1,9 @@
 #!/usr/bin/env python3
 
+print(DeprecationWarning("WARNING: This script is deprecated. Use 'deploy_robot.py' instead. Please remove this script in the future."))
+
+#!/usr/bin/env python3
+
```
Comment (timonegk) [inline scripts/robot_compile.py:6]:
```suggestion
```

Comment (jaagut) [reply scripts/robot_compile.py:6]: This file should be executed. Thus, the shebang should be included

Comment (timonegk) [reply scripts/robot_compile.py:6]: But you don't need two of them.

Comment (jaagut) [reply scripts/robot_compile.py:6]: Oh, sorry. I miss-read `robot_compile` for the `deploy_robots` script, where I just removed one shebang

**Review by timonegk — APPROVED** [review]

**Pull request merged**